sal_interfaces/CCCamera/CCCamera_Events.xml

```diff
@@ -7345,18 +7345,18 @@
     <!--CCSMETA: Array manSerNum indexed by location-->
     <item>
       <EFDB_Name>manSerNum</EFDB_Name>
-      <Description>MISSING</Description>
+      <Description>MISSING (colon delimited array)</Description>
```

Comment:
Can you please change MISSING to something more descriptive? Here for any of the other topics for which you can provide some kind of description.

I assume the data is presently colon-separated? Because if not, we use comma as the separator for everywhere except for lists of paths.

Reply:
HI Russell, these files are generated completely automatically. Where the description is MISSING this indicates that the documentation is either missing in the CCS source, or that there is an issue with the way the description is being transferred. In both cases we are working on fixes, but it will take several release cycles to get them all fixed. 

We agreed long ago to send all camera string arrays as colon delimited, with backslash used as a escape in case the strings contains a colon (or backslash). https://jira.lsstcorp.org/browse/CAP-570. I know the header service relies on this behavior so I would not like to change it.